beancount_import/source/ofx.py

```diff
@@ -617,13 +618,17 @@ def get_securities(soup: bs4.BeautifulSoup) -> List[SecurityInfo]:
 
 
 STOCK_BUY_SELL_TYPES = set(
-    ['BUYMF', 'SELLMF', 'SELLSTOCK', 'BUYSTOCK', 'REINVEST'])
-SELL_TYPES = set(['SELLMF', 'SELLSTOCK'])
+    ['BUYMF', 'SELLMF', 'SELLSTOCK', 'BUYSTOCK', 'REINVEST', 'BUYDEBT',
+     'SELLDEBT', 'SELLOTHER'])
+SELL_TYPES = set(['SELLMF', 'SELLSTOCK', 'SELLDEBT', 'SELLOTHER'])
 
 RELATED_ACCOUNT_KEYS = ['aftertax_account', 'pretax_account', 'match_account']
 
 # Tolerance allowed in transaction balancing.  In units of base currency used, e.g. USD.
 TOLERANCE = 0.05
+UNITPRICE_ERROR_LOWER_BOUND = 0.05
+UNITPRICE_ERROR_UPPER_BOUND = 20.0
```

Comment:
I think I had 5 and 1/5 originally which I thought was already far too conservative. What's the logic for making these choices even more conservative? Do you have a example where the 5 and 1/5 bounds lead to false positives?

Reply:
I don't have an example. Right now I think the only case to trigger this logic is for bond, where the ratio is 100x. So a 20x ratio is enough. I don't object to making it more aggressive.

Reply:
Yeah let's make this 5 and 1/5 then.